src/analysis/inner_product_space/l2_space.lean

```diff
@@ -403,6 +404,22 @@ protected lemma tsum_inner_mul_inner (b : hilbert_basis ι 𝕜 E) (x y : E) :
   ∑' i, ⟪x, b i⟫ * ⟪b i, y⟫ = ⟪x, y⟫ :=
 (b.has_sum_inner_mul_inner x y).tsum_eq
 
+-- Note : this should be `b.repr` composed with an identification of `lp (λ i : ι, 𝕜) 2` with
+-- `pi_Lp 2 (λ i : ι, 𝕜)`, but we don't have this yet (July 2022).
```

Comment:
Could you clarify this TODO to mention that the identification should be between `lp p` and `pi_Lp p`, not just for `p = 2`?